CHANGELOG.next.asciidoc

```diff
@@ -180,6 +180,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...main[Check the HEAD dif
 - Add `xpack.enabled` support for Enterprise Search module. {pull}29871[29871]
 - Add gcp firestore metricset. {pull}29918[29918]
 - Remove strict parsing on RabbitMQ module {pull}30090[30090]
+- Add `kubernetes.container.status.last.reason` metric {issue}27840|[27840]
```

Comment:
```suggestion
- Add `kubernetes.container.status.last.reason` metric {issue}30306|[30306]
```

Reply:
the issue number is 27840, but anyway - changed it to `pull` https://github.com/elastic/beats/pull/30306/commits/28f2f7ec2130b8127fd429126a699fa769874290